feature-libs/product-configurator/rulebased/components/overview-filter-bar/configurator-overview-filter-bar.component.html

```diff
@@ -0,0 +1,58 @@
+<ng-container *ngIf="config?.overview">
```

Comment:
Config is always present
```suggestion
<ng-container *ngIf="config.overview">
```
@Uli-Tiger : Besides this: Why do we need this ng container at all? Don't we know that the OV is always present?

Reply:
first i thought you are right, but if i remove it UI crashes on navigation to OV with
```
ERROR TypeError: ctx.config is undefined
    ConfiguratorOverviewFilterBarComponent_Template configurator-overview-filter-bar.component.html:2
```
maybe we can have a look to it on monday

Reply:
nach dem umbau konnte ich die zeile komplett löschen